airflow/providers/papermill/operators/__init__.py

```diff
@@ -15,3 +15,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from papermill.engines import papermill_engines
+
+from airflow.providers.papermill.operators.papermill import RemoteKernelEngine, REMOTE_KERNEL_ENGINE
+
+papermill_engines.register(REMOTE_KERNEL_ENGINE, RemoteKernelEngine)
```

Comment:
Do we need to register this every-time? Could we register only if we required to use Remote kernel?

Reply:
I am now registering in the kernel hook.